tests/benchmark/read_gbq_colab/dry_run.py

```diff
@@ -38,5 +37,4 @@ def dry_run(*, project_id, dataset_id, table_id, session: bigframes.session.Sess
         project_id=config.project_id,
         dataset_id=config.dataset_id,
         table_id=config.table_id,
-        session=config.session,
```

Comment:
probably need to pass the global session for these. But we don't want to fetch the global session too early. 🤔  Maybe need to update `get_execution_time` to use the global session if none was supplied.